commands.py

```diff
@@ -0,0 +1,222 @@
+import asyncio
+from asyncio.tasks import Task
+from typing import Any, Awaitable, Generator, List, Optional
+from weakref import WeakSet, ref
+
+import sublime
+import sublime_plugin
+
+from .globalstate import call_soon_threadsafe, run_coroutine
+import contextlib
```

Comment:
Is there a rule to the grouping and order? if so then it should probably be enforced somehow or it will be easy to get wrong.

Reply:
> Is there a rule to the grouping and order? if so then it should probably be enforced somehow or it will be easy to get wrong.

I'm just calling `black .` and `isort .` on the command-line. Do you expect to see a basic CI?